app/presenters/conviction_result_presenter.rb

```diff
@@ -1,4 +1,6 @@
 class ConvictionResultPresenter < ResultsPresenter
+  include ConvictionDecorator
```

Comment:
This is not needed Tony as you don't access through the presenter, but instead through the `conviction_type`, which is the one already including the decorator.